lndclient/lnd_services.go

```diff
@@ -39,6 +39,7 @@ type LndServices struct {
 	Signer        SignerClient
 	Invoices      InvoicesClient
 	Router        RouterClient
+	Versioner     VersionerClient
```

Comment:
Does it need to be exported? We can store the version info when we retrieve it on startup and expose that. It won't ever change, so there is no need to re-retrieve?

Reply:
See comment above.